SystemManager/Src/SystemManager.cpp

```diff
@@ -33,28 +34,24 @@ SystemManager::~SystemManager() {}
 
 void SystemManager::flyManually() {
     for(;;){
+
         this->rcInputs_ = rcController_->GetRCControl();
 
-        //TO-DO: need to implement it using is_Data_New;
-        // boolean is true if data has not changed since the last cycle
-        bool is_unchanged{
-        	rcInputs_.throttle == prevthrottle &&
-        	rcInputs_.yaw == prevyaw &&
-			rcInputs_.roll == prevroll &&
-			rcInputs_.pitch == prevpitch
-        };
+        printf("Arm: %f\r\n", this->rcInputs_.arm);
```

Comment:
I would take this out of our comitted code, or at the very least hide it under a IFDEF DEBUG flag

Reply:
Okay sounds good. I can meet up tomorrow at around 2:30 PM.